solidity/test/base/Transcript.t.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: UNLICENSED
+// This is licensed under the Cryptographic Open Software License 1.0
+pragma solidity ^0.8.28;
+
+import {Transcript} from "../../src/base/Transcript.sol";
+import {MODULUS, WORD_SIZE, FREE_PTR} from "../../src/base/Constants.sol";
+
+library TranscriptTest {
+    function testWeCanDrawChallenge() public pure {
+        uint256[1] memory transcriptAPtr = [uint256(12345)];
```

Comment:
you can do something like: 
using Transcript for uint256[1];

then: 
uint256 challengeA1 = transcriptAPtr.drawChallenge();